packages/sdk/src/internal/payloads/payloads.ts

```diff
@@ -405,3 +407,63 @@ export type DialogResponse = Payload & {
 	submitted: boolean;
 	text?: string;
 };
+
+/**
+ * @hidden
+ * Sanitize network input
+ * @param payload A payload fresh from the websocket
+ */
+export function processPayload(payload: Partial<any>) {
+	if (!payload) { return payload; }
+
+	// payload guids
+	if (payload.actorId) { payload.actorId = parseGuid(payload.actorId); }
```

Comment:
I'm afraid this isn't a viable solution. This code creates a maintenance problem. We need a solution that doesn't require manual fixup whenever a payload is added or changed. See my JSON Schema suggestion. Also, I don't think we need to tackle this now. We have a workaround.